lib/bolt/project_manager.rb

```diff
@@ -195,7 +195,7 @@ def migrate
 
       migrator.migrate(
         @config.project,
-        @config.modulepath
+        @config.modulepath[0...-1]
```

Comment:
```suggestion
        @config.modulepath[0..-1]
```
I'm not even sure what `...` does, but this results in `modulepath` being removed from `bolt-project.yaml` entirely, but *only* if it was either `[modules]` or `[modules, site-modules, site]`.

Reply:
It makes sure we drop `.modules/` from the modulepath, since `Bolt::Config.modulepath` automatically appends it to the default modulepath _and_ the configured modulepath.

The current behavior for the migrator is to remove `modulepath` from `bolt-project.yaml`, so that's expected behavior...